### DISCUSSION TITLE
Files reprocess after every reboot

### DISCUSSION THREAD

**Discussion by string97bean** [AzuraCast/AzuraCast#5423] [Q&A]
Not sure if this is an expected behavior, but I have a large library that is connected via an SMB share through a docker override file directly to my station.  Everytime I have to restart the azuracast service it reprocesses all of the files and it takes days.  Is this normal?

**Comment by ghost** [comment]
Normal can be different based on your setup
is the SMB share connection in the same data center? or connected through internet? 
if connected via the internet what is the bandwidth and how reliable is it? 
how large is your library?
Could you separate your library by genre?
